docs/reference/quickstart/getting-started.asciidoc

```diff
@@ -193,31 +222,100 @@ You should receive a response indicating there were no errors.
 ===============
 
 [discrete]
-[[qs-search-data]]
+[[getting-started-mappings-and-data-types]]
+=== Define mappings and data types
+
+<<elasticsearch-intro-documents-fields-mappings,Mappings>> define how data is stored and indexed in {es}, much like a  schema for a relational database.
+
+[discrete]
+[[getting-started-dynamic-mapping]]
+==== Use dynamic mapping
+
+When using dynamic mapping, {es} automatically creates mappings for new fields by default.
+The documents we've added so far have used dynamic mapping, because we didn't specify a mapping when creating the index.
+
+Let's add a new document to the `books` index with a field that doesn't exist, to see how dynamic mapping works.
+
+[source,console]
+----
+POST /books/_doc
+{
+  "name": "The Great Gatsby",
+  "author": "F. Scott Fitzgerald",
+  "release_date": "1925-04-10",
+  "page_count": 180,
+  "new_field": "This is a dynamically added field"
+}
+----
+// TEST[continued]
+
+View the mapping for the `books` index. The new field `new_field` has been added to the mapping with a `text` data type.
+
+[source,console]
+----
+GET /books/_mapping
```

Comment:
Your previous step says "see how dynamic mapping works" but you dont show a result. This again may be an opportunity to show a result as to what happened?

Reply:
resolve this because we talk about examples in general in another [comment](https://github.com/elastic/elasticsearch/pull/113558#discussion_r1778178420)